x-pack/plugins/cloud_security_posture/server/routes/compliance_dashboard/compliance_dashboard.ts

```diff
@@ -52,16 +59,21 @@ export const defineGetComplianceDashboardRoute = (router: CspRouter): void =>
           keep_alive: '30s',
         });
 
+        const policyTemplate = request.params.policy_template as PosturePolicyTemplate;
+
         const query: QueryDslQueryContainer = {
-          match_all: {},
+          bool: {
+            // TODO: CIS AWS - replace filtered field to `policy_template` when available
+            filter: [{ term: { 'rule.benchmark.id': policyTemplate } }],
+          },
```

Comment:
IMO it's be easier to keep track of things if for the `rule.benchmark.id` we'd pass a benchmark id and not a policy template. then change that when we do. right now it reads very wrong. there is no `benchmark.id` of `cspm` or `kspm`, but that's the value `policyTemplate` should have, instead we pass an `input["type"]`, so when i read `policyTemplate` passed to a `benchmark.id` filter i know the value of `policyTemplate` is not `benchmark.id`

Reply:
discussed it a bit with @kfirpeled as well since i myself wasn't exactly sure what is the best way to handle this in the first place. in general you are correct, but we arrived at the conclusion that since this code is extremely temporary and most like that i will be the only one that is taking effect from that part. its currently not worth wasting time over changing those parts as it will then later need to be reverted back. hopefully by the next week or two its no longer a problem